version/VERSION

```diff
@@ -1 +1 @@
-1.11.0
\ No newline at end of file
+1.11.0-alpha
```

Comment:
Should we update this manually? I guess this is because it's an alpha?

Reply:
Yes - if we don't update it the version will register as 1.11.0 in CRT.